Source/JavaScriptCore/runtime/IntlDurationFormat.cpp

```diff
@@ -239,7 +239,8 @@ void IntlDurationFormat::initializeDurationFormat(JSGlobalObject* globalObject,
         }
     }
 
-    m_fractionalDigits = intlNumberOption(globalObject, options, vm.propertyNames->fractionalDigits, 0, 9, 0);
+    constexpr unsigned undefinedValue = UINT32_MAX;
```

Comment:
We should use `std::numeric_limits<unsigned>::max()` instead of `UINT32_MAX`. Also, you can just define this once at the top of the file, like [this](https://github.com/WebKit/WebKit/blob/main/Source/JavaScriptCore/runtime/TemporalDuration.cpp#L38).

Reply:
Thank you for the review, I've fixed!